CMakeLists.txt

```diff
@@ -67,6 +67,40 @@ if (F3D_USE_EXTERNAL_NLOHMANN_JSON)
   find_package(nlohmann_json REQUIRED)
 endif ()
 
+# check if we need to link with libatomic (not needed on MSVC)
+if ((NOT Windows) AND (NOT APPLE))
```

Comment:
Maybe we should test if the compiler is GCC.  
We had a similar issue where sometimes we had to link explicitly with `filesystem`.  
Maybe we should do something like this for it too.

Reply:
Here's what we are doing for filesystem: https://github.com/f3d-app/f3d/blob/b40c3c4b05e8db307642194878d10e7e2132134e/application/CMakeLists.txt#L33  
Simpler but not automatic, so maybe your suggestion is better.  
Wdyt @mwestphal ?

Reply:
I like simpler, manual solution, especially for a usecase as specific as this one.